src/main/groovy/io/seqera/wave/service/k8s/K8sService.groovy

```diff
@@ -33,7 +33,7 @@ import io.seqera.wave.configuration.SpackConfig
  */
 interface K8sService {
 
-    enum JobStatus { Pending, Running, Succeeded, Failed }
+    enum JobStatus { PENDING, RUNNING, SUCCEED, FAILED, UNKNOWN }
```

Comment:
Can the `JobStatus` remain as it was before?

Reply:
currently job is only used by transfer, so here we have opportunity to standardize it and make it same as you did for transfer. But If you want to keep the old one i can add mapping between job and transfer status

Reply:
> But If you want to keep the old one i can add mapping between job and transfer status

Yes, let's not mix different things